cypress/integration/home_page_spec.js

```diff
@@ -6,29 +6,31 @@ describe('The Home Page', function() {
 
 describe('The Login Page', function () {
     afterEach(() => {
-    cy.request({
-        url: '/api/reset-server',
-        method: 'GET'
-    });
-    cy.visit('/');
+        cy.request({
+            url: '/api/reset-server',
+            method: 'GET'
+        });
+        cy.visit('/');
     })
+
     it('sets auth cookie when logging in via form submission', function () {
-    // destructuring assignment of the this.currentUser object
-    const username = 'jane'
+        // destructuring assignment of the this.currentUser object
```

Comment:
there's no destructuring assignment here :P

Reply:
this is from another pr and is already on develop, I just merged develop into mine